app/views/catalog/_document.html.erb

```diff
@@ -1,4 +1,4 @@
 <% # container for a single doc -%>
-<% view_config = local_assigns[:view_config] || blacklight_config.view_config(document_index_view_type) %>
-<% document_component = blacklight_config.view_config(:show).document_component -%>
+<% view_config = local_assigns[:view_config] || blacklight_config.view_config(document_index_view_type, action_name: action_name) %>
+<% document_component = view_config.document_component -%>
```

Comment:
Are you able to write test for this behavior?  It looks like it was inadvertently reverted because there was no test here before. Are you able to write one?